docker/entrypoint-webapp.sh

```diff
@@ -1,14 +1,8 @@
 #!/bin/sh
 set +ex
 
-# Make these available via Settings in the app
-export SETTINGS__GIT_COMMIT="$APP_GIT_COMMIT"
-export SETTINGS__BUILD_DATE="$APP_BUILD_DATE"
-export SETTINGS__GIT_SOURCE="$APP_BUILD_TAG"
-
-
-rails db:migrate
-rails data:migrate
+bundle exec rails db:migrate
```

Comment:
Seeing as the migration is done here, I don't think the rails-migrations jobs are needed, so probably should be removed the deploy instructions.